packages/nextjs/pages/builders/0x38D9cFf58D233AF0B9c1434EEDE012009D23c971.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+import { Address } from "~~/components/scaffold-eth";
+
+const socials = [
+  {
+    link: "https://github.com/miralsuthar",
+    title: "github",
+  },
+  {
+    link: "https://twitter.com/miral182000",
+    title: "twitter",
+  },
+  {
+    link: "https://miralst.xyz",
+    title: "website",
+  },
+];
+
+const ProfilePage = () => {
+  return (
+    <section className="flex justify-center items-center mt-10">
+      <div className="border flex gap-6 flex-col w-96 items-center border-white rounded-md p-5">
+        <Address
+          address="0x38D9cFf58D233AF0B9c1434EEDE012009D23c971"
+          size="3xl"
+          disableAddressLink={false}
+          format="short"
```

Comment:
We can remove this two props, since their default values are same what we are passing here 🙌

https://docs.scaffoldeth.io/components/Address#props